price_parser/parser.py

```diff
@@ -25,7 +25,8 @@ def amount_float(self) -> Optional[float]:
 
     @classmethod
     def fromstring(cls, price: Optional[str],
-                   currency_hint: Optional[str] = None) -> 'Price':
+                   currency_hint: Optional[str] = None,
+                   decimal_separator_hint: Optional[str] = None) -> 'Price':
```

Comment:
What do you think about renaming it to `decimal_separator`? The difference with currency_hint is that currency_hint is just a hint and can be ignored, while here it is a hard override.

Reply:
yeah, it makes sense